cirq/experiments/google_v2_supremacy_circuit_test.py

```diff
@@ -21,10 +21,27 @@ def test_google_v2_supremacy_circuit():
         ops.HGate))) == 40
     qubits = [GridQubit(i, j) for i in range(4)
               for j in range(5)]
+    assert isinstance(circuit.operation_at(qubits[0],2).gate, ops.RotYGate)
+    assert isinstance(circuit.operation_at(qubits[1],2).gate, ops.RotYGate)
+    assert isinstance(circuit.operation_at(qubits[8],2).gate, ops.RotXGate)
     assert circuit.operation_at(qubits[0], 1).gate == ops.CZ
     assert circuit.operation_at(qubits[5], 2).gate == ops.CZ
     assert circuit.operation_at(qubits[8], 3).gate == ops.CZ
     assert circuit.operation_at(qubits[13], 4).gate == ops.CZ
     assert circuit.operation_at(qubits[12], 5).gate == ops.CZ
     assert circuit.operation_at(qubits[13], 6).gate == ops.CZ
     assert circuit.operation_at(qubits[14], 7).gate == ops.CZ
+
+
+def test_google_v2_supremacy_bristlecone():
```

Comment:
Do another test using fewer than the maximum number of rows.

Reply:
Done